notebooks/community/vertex_endpoints/torchserve/dreambooth_stablediffusion.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
<p>The links need to be updated for the new location.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Done.